### ISSUE TITLE
Y isn't sorted properly

### ISSUE THREAD

**Issue by lotsofs** [dwyl/english-words#78]
Some words with the letter Y appear under the letter I instead.

Examples:
NYET appears in between NIESHOUT and NIETZSCHE

ABIRRITATIVE is followed by
ABYS
ABYSM
ABYSMAL
[ etc ]
before continuing with
ABYSSUS
ABISTON
ABIT

XXV -> XXX -> Z -> ZA -> ZABAEAN

Words like YACHT, YACCA, YARTH etc all appear under I

Did you perhaps use the Frisian alphabet when sorting the list?

**Comment by RUGMJ7443** [comment]
if you're in javascript you can use `.sort` to sort them alphabetically
Hope this helps

**Comment by JayArrrgh** [comment]
> if you're in javascript you can use `.sort` to sort them alphabetically

True, but even the shorter 370k-word "alpha" version can take about 30 seconds to sort, so you wouldn't want to do it at runtime, you'd want to save the result as a new file - and make it available here.

So that's what I'll do! List.Sort() and System.IO.File.WriteAllLines() provided this result in UTF-8 (same as original words_alpha.txt, I believe) with what looks to me like the same CRLF line endings. I had to manually delete one extraneous CRLF at the end of the file but otherwise this is just as C# outputted it.

[words_alpha_sorted.txt](https://github.com/dwyl/english-words/files/5906472/words_alpha_sorted.txt)
